app/javascript/dashboard/assets/scss/_helper-classes.scss

```diff
@@ -51,7 +51,12 @@
   }
 }
 
-.justify-space-between {
+.flex-view {
+  display: flex;
+}
+
+.flex-space-between {
+  display: flex;
```

Comment:
This doesn't need to be here.  Instead we could `flex-view flex-space-between` where it is being used.

Reply:
fixes by https://github.com/chatwoot/chatwoot/pull/4061/commits/8ae236b568fa2cd316395131f5222bb81f5743e6